ReactWindows/ReactNative/UIManager/DynamicAutomationPeer.cs

```diff
@@ -0,0 +1,56 @@
+using System.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
+
+namespace ReactNative.UIManager
+{
+    /// <summary>
+    /// Automation peer that allows to dynamically change behavior.
+    /// </summary>
+    /// <typeparam name="T">Type of owner of the automation peer.</typeparam>
+    public sealed class DynamicAutomationPeer<T> : FrameworkElementAutomationPeer, IInvokeProvider
```

Comment:
>DynamicAutomationPeer [](start = 24, length = 21)

Can we put this, AccessibilityHelper, AccessibilityTapEvent, ImportantForAccessibility, IAccessible, and AccessibilityTrait all in a single folder / namespace? At some point, we need to start using the "code owners" feature of GitHub, and having this in one place will help.

Reply:
@rozele I went ahead and moved everything in ReactNative project in Accessibility folder. Also moved everything in ReactNative.Accessibility namespace

---
In reply to: [170328497](https://github.com/Microsoft/react-native-windows/pull/1645#discussion_r170328497) [](ancestors = 170328497)